.changeset/beige-olives-beam.md

```diff
@@ -0,0 +1,7 @@
+---
+"@directus/storage-driver-s3": patch
+"@directus/storage": patch
+"tests-blackbox": patch
+---
+
+Reused S3 client and increased maxSockets
```

Comment:
"Reused..." sounds as if it is actually no longer the case.
I would probably put it something like this:
```suggestion
Addressed S3 storage scaling issues by using a customized request handler
```

Reply:
Sounds good, thanks! 👍